skypy/galaxies/tests/test_velocity_dispersion.py

```diff
@@ -0,0 +1,25 @@
+from skypy.galaxies import velocity_dispersion
+import numpy as np
+
+
+def test_schecter_vdf():
+    # test schecter velocity dispersion function
+    vd_min = 100
+    vd_max = 200
+    size = 100
+    samples = velocity_dispersion.schecter_vdf(vd_min, vd_max, size=size)
+
+    # test output is within limits and sizee
+    assert np.all(samples <= vd_max)
+    assert np.all(samples >= vd_min)
+    assert np.sum((vd_min <= samples) & (samples <= vd_max)) == size
+
+    # test output shape when scale is scalar
```

Comment:
Scale should not be an input parameter. Replace this test for the shape with shape tests as done [here](https://github.com/skypyproject/skypy/blob/df2536974c122dd72498f8d146913c14c832833b/skypy/galaxies/tests/test_stellar_mass.py#L23) and [here](https://github.com/skypyproject/skypy/blob/df2536974c122dd72498f8d146913c14c832833b/skypy/galaxies/tests/test_luminosity.py#L7).